package.json

```diff
@@ -5,9 +5,9 @@
     "build": "pulp build --censor-lib --strict"
```

Comment:
Hum.., tried this before but it didn't work, pushed anyway but build is still failing.

I'm having problems building this locally, seems to be related to `pulp` (FWIW, I don't like this build tool, using my own Makefiles for my projects).